src/apps/testapps/testPolyfill.c

```diff
@@ -262,6 +231,35 @@ TEST(polyfillTransmeridian) {
     free(hexagonsTMH);
 }
 
+TEST(polyfillTransmeridianComplex) {
+    // This polygon is "complex" in that it has > 4 vertices - this
+    // tests for a bug that was taking the max and min longitude as
+    // the bounds for transmeridian polygons
+    GeoCoord verts[] = {{0.1, -M_PI + 0.00001},  {0.1, M_PI - 0.00001},
+                        {0.05, M_PI - 0.2},      {-0.1, M_PI - 0.00001},
+                        {-0.1, -M_PI + 0.00001}, {-0.05, -M_PI + 0.2}};
+
+    Geofence geofence;
```

Comment:
Why not `Geofence geofence = { verts, 6 };`?

Reply:
Will update using designated initializers.